js/repl/ReplOptions.js

```diff
@@ -46,12 +46,15 @@ type Props = {
   runtimePolyfillState: PluginState,
 };
 
-const ReplOptions = (props: Props) =>
+const ReplOptions = (props: Props) => (
   <div className={`${styles.wrapper} ${props.className}`}>
-    {props.isExpanded
-      ? <ExpandedContainer {...props} />
-      : <CollapsedContainer {...props} />}
-  </div>;
+    {props.isExpanded ? (
+      <ExpandedContainer {...props} />
+    ) : (
+      <CollapsedContainer {...props} />
+    )}
+  </div>
+);
```

Comment:
Are these formatting changes all from Prettier? 😄

Reply:
Yeah, looks like it bumped. I'll revert and push to a different PR.

Reply:
No worries. I don't mind being pragmatic about it, so long as we also bump the Prettier version in `yarn.lock` so we aren't undoing each others changes on PRs.